include/picongpu/simulation_defines/param/ionizer.param

```diff
@@ -447,8 +447,25 @@ namespace thomasFermi
      * @note This cutoff value should be set in accordance to FLYCHK calculations,
      *       for instance! It is not a universal value and requires some preliminary
      *       approximations!
+     *
+     * unit: 1 / m^3
+     *
+     * default: ion number density = critical electron number density for lambda_laser = 800nm
```

Comment:
1) title: lower +ion+ density cutoff

"default" is a bit misleading here.
Maybe write:
```
example: 1.7422e27 as a hydrogen ion number density equal to the corresponding critical electron number density for a 800nm laser
```
```